lib/bambora/client.rb

```diff
@@ -115,5 +151,13 @@ def json_client
         sub_merchant_id: sub_merchant_id,
       )
     end
+
+    def www_form_client
+      @www_form_client ||= Bambora::WWWFormClient.new(
+        base_url: base_url,
```

Comment:
I am going to leave this one as is because using a hash in place of key word arguments will be deprecated in 2.7. I don't want to block support for future versions of ruby.

https://bugs.ruby-lang.org/issues/14183

Reply:
Ah, I just realized that the Clients use an options hash and not key word args, so my comment about Ruby 2.7 doesn't apply here. I'll consider moving this options hash to a method in a future PR.